lib/zendesk_apps_support/package.rb

```diff
@@ -72,6 +72,15 @@ def template_files
       files.select { |f| f =~ /^templates\/.*\.hdbs$/ }
     end
 
+    def no_template
+      manifest = manifest_json
```

Comment:
I don't quite get it. Why not use `manifest_json` directly? No change will be made to it in this method as far as I can see.